src/views/Requests/ReviewTabs.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+
+import {
+    Spinner,
+    Tab,
+    TabList,
+    TabPanel,
+    Tabs
+} from '@impact-market/ui';
+
+import Communities from './Communities'
+import String from '../../libs/Prismic/components/String';
+
+const ReviewTabs = ({ communities, loading, reviewsByCountry, numberOfCommunitiesByReview, myCountrySelected, userCountry, setReview } : any) => {    
+
+    const [reviews] = useState(['pending', 'claimed', 'declined']);
+
+    //  Get how many communties there are in each review (pending, claimed, declined)
+    const numberOfEachReview = (review: any) => {
+        let otherCountriesNumberOfReviews = 0
+        let myCountryNumberOfReviews = {} as any
+
+        !!myCountrySelected ?
+            !!Object.keys(reviewsByCountry).length && (
+                reviewsByCountry.filter((communitiesNumber: any) => 
+                    userCountry === communitiesNumber?.country).map((quantity: any) => (
+                        myCountryNumberOfReviews = quantity
+                    )) 
+            )
+        :
+            !!Object.keys(numberOfCommunitiesByReview).length && (
+                numberOfCommunitiesByReview.filter((reviewName: any) => reviewName.review === review).map((count: any) => (
+                    otherCountriesNumberOfReviews = parseInt(count.count, 10)
+                ))
+            )
+
+        return !!myCountrySelected ? parseInt(myCountryNumberOfReviews[review], 10) : otherCountriesNumberOfReviews
+    }
```

Comment:
I don't understand this function.

When the country is selected you set the last position of the filtered results, as `myCountryNumberOfReviews` but then you treat it as an obj or arr?

Reply:
`reviewsByCountry` returns objects with all countries and review status of each one, like so:

```
accepted: "1"
claimed: "0"
count: "1"
country: "BR"
declined: "0"
pending: "0"

accepted: "1"
claimed: "0"
count: "1"
country: "PT"
declined: "0"
pending: "0"
```
etc...

I run a .filter in `reviewsByCountry` to return only the object of user's country and assign it to `myCountryNumberOfReviews`.

At the end of the function I return the value of the key `[review]` (for example, pending) from the `myCountryNumberOfReviews` object, and transform it to a number like so: `parseInt(myCountryNumberOfReviews[review], 10)`